static/login.html

```diff
@@ -0,0 +1,42 @@
+<!doctype html>
+
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+     <link rel="stylesheet" href="http://fonts.googleapis.com/css?family=Roboto:300,400,500,700" type="text/css">
+    <title>Login</title>
+    <style>
```

Comment:
I could do it but I would need to duplicate the css and I was not really in favor of that. Do you have any idea for not duplicating it? The only way I thought would be to have have this page under the src folder and be an AMP page so that the relevant css could be added via the template. Any idea would be appreciated!